semantic.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Div soup page</title>
+</head>
+
+<body>
+ <!-- ovo je semanticki napisan kod koji ce jos sadrazavat i komentare  -->
+ <!-- ovo je Header dio koji sadrzi i  logo, naziv i navigaciju  -->
+ <header>
+  <img src="html logo.png" alt="html logo" width="150px">
+  <h1>Ovo je Semantic html stranica</h1>
+  <nav>
+   <a href="">Home</a>
```

Comment:
@borissimic Prazan `href` će refresh-at stranicu, nadam se da je ovo napisano s namjerom 😄

Reply:
namjerno je ostavljen tako , to je zato sta bi napravio nav bar  fixated kroz css pa kad stisnes home da  te vrati na pocetak stranice ... sad neznam dali je bolje da ostavim prazno ili da u href stavim "semantic.html"